Add draft deliverable 2.6 feedback to revision of Cybersecurity Act (#127)
orcwg/orcwg: cyber-resilience-sig/coordination/enisa/deliverable-2-6.md

* We recognise the importance of increasing EU resources to ENISA as part of CSA review. Notably, ENISA’s role as a CVE Numbering Authority (CNA) is important and industry welcomes ENISA’s goal to become a Root CNA.
* MITRE’s role as Secretariat plays a key role to provide a resource to boost supply chain resilience.  Whilst it is regrettable the recent publicity and uncertainty in funding we are pleased that this matter has now been partially resolved.  
* The momentary alarm did however serve as a timely reminder to both governments and industry as to CVE’s global importance and need for wider  global support. The establishment of [the CVE foundation](https://www.thecvefoundation.org/) provides ever greater visibility in supporting this crucial namespace pillar of global supply chain resilience.
* Indeed, this has also been recognised by several European governments who subsequently confirmed their wish to support (technically and financially) and thus we would hope that this support can be mirrored at an EU level, perhaps via ENISA, into a system which is mirrored infrastructure in Europe and co-developed.

> * We also see the growing role of national Computer Security Incident Response Team (CSIRTs), such as the National Cyber Security Centre (NSSC) in Ireland, largely supporting bug bounty programs, Capture the Flag (CTF) and other activities with local academia, government and businesses, investing in vulnerability discovery and management, and thus important to double down on the CVE program in a uniformed, cross-European and international way.

Reviewer: It seems like that should be "NCSC Ireland", as indicated at https://www.ncsc.gov.ie/ too.

Author: Is the typo fix all you had in mind here?

> * We also see the growing role of national Computer Security Incident Response Team (CSIRTs), such as the National Cyber Security Centre (NSSC) in Ireland, largely supporting bug bounty programs, Capture the Flag (CTF) and other activities with local academia, government and businesses, investing in vulnerability discovery and management, and thus important to double down on the CVE program in a uniformed, cross-European and international way.

Author: Is fixing the typo like this the only change you were suggesting?

```suggestion
* We also see the growing role of national Computer Security Incident Response Team (CSIRTs), such as the National Cyber Security Centre (NCSC) in Ireland, largely supporting bug bounty programs, Capture the Flag (CTF) and other activities with local academia, government and businesses, investing in vulnerability discovery and management, and thus important to double down on the CVE program in a uniformed, cross-European and international way.
```

> * It would also be advisable for the European Commission to include this need for reinforced collaboration in their next inter-institutional dialogue with CISA.

Reviewer: Might help to refer to it as something like "USA CISA" for purposes of disambiguation.

> * Whilst we recognise the importance of NIS2’s European Vulnerability Database (EUVD) we feel there is still room for better alignment and co-ordination. Indeed, the EUVD’s beta version includes identifiers which are not aligned with CVEs or easily mapped.  This poses a significant risk to organisations and companies defending critical networks versus criminal or state sponsored hackers due to similar patterns but dissimilar meanings. This will create chaos for all producers and consumers of software bills of materials (SBOMs) and of vulnerability and exploitation data.  

Reviewer: It also increases the reporting burden on software producers if they end up needing to apply for both CVE and EUVD assignments on each disclosed vulnerability.

> * Whilst we recognise the importance of NIS2’s European Vulnerability Database (EUVD) we feel there is still room for better alignment and co-ordination. Indeed, the EUVD’s beta version includes identifiers which are not aligned with CVEs or easily mapped.  This poses a significant risk to organisations and companies defending critical networks versus criminal or state sponsored hackers due to similar patterns but dissimilar meanings. This will create chaos for all producers and consumers of software bills of materials (SBOMs) and of vulnerability and exploitation data.  

Author: ```suggestion
* Whilst we recognise the importance of NIS2’s European Vulnerability Database (EUVD) we feel there is still room for better alignment and co-ordination. Indeed, the EUVD’s beta version includes identifiers which are not aligned with CVEs or easily mapped.  This poses a significant risk to organisations and companies defending critical networks versus criminal or state sponsored hackers due to similar patterns but dissimilar meanings. This will create chaos for all producers and consumers of software bills of materials (SBOMs) and of vulnerability and exploitation data. It also increases the reporting burden on software producers if they end up needing to apply for both CVE and EUVD assignments on each disclosed vulnerability.
```

---
SIG: Cyber Resilience SIG
Document Type: Deliverable
Number: 2.6
Status: ⚠️ Draft
Group: ENISA
Date: 2025-06-14
---

# Feedback to call for evidence on Cybersecurity Act Revision

## Context

> Provide feedback to the [EU Commission][]'s [open call for evidence](https://ec.europa.eu/info/law/better-regulation/have-your-say/initiatives/14578-The-EU-Cybersecurity-Act_en) as part of its [initiative to revise the Cybersecurity Act (CSA)](https://digital-strategy.ec.europa.eu/en/news/commission-opens-consultation-revising-eu-cybersecurity-act). Urge for a stronger technical and strategic role for ENISA in light of the [recent funding issues of the US CVE program](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures#2025_funding_issues) and the increased importance of vulnerability handling in EU legislation.

Author: This context ***for us*** and won't be shared in the feedback itself. I should have made this more clear.

## Goals

* Urge the EU Commission to increase ENISA's mandate regarding CVE handling
* Avoid disproportionate reliance on U.S. government funding for CVEs

## Feedback

In light of the [CSA review](https://digital-strategy.ec.europa.eu/en/news/commission-opens-consultation-revising-eu-cybersecurity-act) and wider need for harmonisation of vulnerability disclosure requirements across several EU regulations (e.g. CRA, NIS2, DORA, AI act) as well as further sharing of supply-chain best-practices, we wish to share and reiterate the importance of the Common Vulnerabilities and Exposures (CVE) system and a stronger technical and strategic role for ENISA given its growing international objective:

> * We recognise the importance of increasing EU resources to ENISA as part of CSA review. Notably, ENISA’s role as a CVE Numbering Authority (CNA) is important and industry welcomes ENISA’s goal to become a Root CNA.

Reviewer 2: I would add: We also believe that ENISA should step up as a  CNA of Last Resource (for Europe, regardless of report or vendors jurisdiction). Secondly we believe that ENISA should contribute systematically and long term to the CVE system as a whole; in general, and in particular around governance, establishing industry best practices and driving improvements. This also includes a pro-active, operational role, in ensuring that coordinated disclosure processes are followed and that the data is of sufficient quality to be useful in a European setting in general and for compliance with, for example, NIS2 and the CRA in particular.  This may, at times, include helping the industry by annotating records, defining vocabulary or by re-publishing advisories after normalisation.

Author: Adressed in https://github.com/orcwg/orcwg/pull/127/commits/0cf2687e37279f46dda02c27881011e0e272a357.

* MITRE’s role as Secretariat plays a key role to provide a resource to boost supply chain resilience.  Whilst it is regrettable the recent publicity and uncertainty in funding we are pleased that this matter has now been partially resolved.  

> * The momentary alarm did however serve as a timely reminder to both governments and industry as to CVE’s global importance and need for wider  global support. The establishment of [the CVE foundation](https://www.thecvefoundation.org/) provides ever greater visibility in supporting this crucial namespace pillar of global supply chain resilience.

Author: Would that work?

```suggestion
* The momentary alarm did however serve as a timely reminder to both governments and industry as to CVE’s global importance for supply chain resilience and need for wider global support.
```

> * The momentary alarm did however serve as a timely reminder to both governments and industry as to CVE’s global importance and need for wider  global support. The establishment of [the CVE foundation](https://www.thecvefoundation.org/) provides ever greater visibility in supporting this crucial namespace pillar of global supply chain resilience.

Reviewer 2: Not at ease with mentioning/recommending the CVE Foundation in this broad a context.

If you want to mention it - I would say something like:

> "While establishment of [the CVE foundation](https://www.thecvefoundation.org/) is potentially a good first step to cover the operational, day to day, running, of the CVE system -- it is not a replacement for the roles that CISA, NIST, JPCERT, ENISA et.,al. currently play. These entities, as both a representative of the regulator and the closest thing to representing the citizens and public sector, are key to ensuring a fair, global and long term approach - balancing the interests of the reporters, the industry and the consumers of software.

Author: @oej made the same comment. Happy to remove mention altogether.

* It would also be advisable for the European Commission to include this need for reinforced collaboration in their next inter-institutional dialogue with USA CISA.
* Whilst we recognise the importance of NIS2’s European Vulnerability Database (EUVD) we feel there is still room for better alignment and co-ordination. Indeed, the EUVD’s beta version includes identifiers which are not aligned with CVEs or easily mapped.  This poses a significant risk to organisations and companies defending critical networks versus criminal or state sponsored hackers due to similar patterns but dissimilar meanings. This will create chaos for all producers and consumers of software bills of materials (SBOMs) and of vulnerability and exploitation data. It also increases the reporting burden on software producers if they end up needing to apply for both CVE and EUVD assignments on each disclosed vulnerability.

> 

Author: Any chance you could reframe this in a couple of sentences? We only have 200-300 characters left, we can shorten other text a bit, but there isn't that much room avaible.

> 

Author: @UlisesGascon: Any chance you could reframe this in a couple of sentences? We only have 200-300 characters left, we can shorten other text a bit, but there isn't that much room available.

> 

Reviewer 3: Now is 259 characters :+1:

---
SIG: Cyber Resilience SIG
Document Type: Deliverable
Number: 2.6
Status: ⚠️ Draft
Group: ENISA
Date: 2025-06-14
---

# Feedback to call for evidence on Cybersecurity Act Revision

## Context

Provide feedback to the [EU Commission][]'s [open call for evidence](https://ec.europa.eu/info/law/better-regulation/have-your-say/initiatives/14578-The-EU-Cybersecurity-Act_en) as part of its [initiative to revise the Cybersecurity Act (CSA)](https://digital-strategy.ec.europa.eu/en/news/commission-opens-consultation-revising-eu-cybersecurity-act). Urge for a stronger technical and strategic role for ENISA in light of the [recent funding issues of the US CVE program](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures#2025_funding_issues) and the increased importance of vulnerability handling in EU legislation.

## Goals

* Urge the EU Commission to increase ENISA's mandate regarding CVE handling
* Avoid disproportionate reliance on U.S. government funding for CVEs

## Draft feedback

> [!NOTE]
> Feedback must be provided in a comment box (so no formatting option). Text length is limited to 4,000 characters.

> > In light of the and wider need for harmonisation of vulnerability disclosure requirements across several EU regulations (e.g. CRA, NIS2, DORA, AI act) as well as further sharing of supply-chain best-practices, we wish to share and reiterate the importance of the Common Vulnerabilities and Exposures (CVE) system and a stronger technical and strategic role for ENISA given its growing international objective:

Reviewer 2: That is fine - was more to suggest another angle as to make it clear why ORC cares (and the EC should) care on this issue which they may see as totally unrelated.

I.e. repair the lack of a strategic long term thing on their end into which this fits./

> 
> We recognise the importance of increasing EU resources to ENISA as part of CSA review. Notably, ENISA’s role as a CVE Numbering Authority (CNA) is important and industry welcomes ENISA’s goal to become a Root CNA.
> 
> We also believe that ENISA should step up as a CNA of Last Resort (for Europe, regardless of report or vendors jurisdiction). Secondly we believe that ENISA should contribute systematically and long term to the CVE system as a whole; in general, and in particular around governance, establishing industry best practices and driving improvements. This also includes a pro-active, operational role, in ensuring that coordinated disclosure processes are followed and that the data is of sufficient quality to be useful in a European setting in general and for compliance with, for example, NIS2 and the CRA in particular. This may, at times, include helping the industry by annotating records, defining vocabulary or by re-publishing advisories after normalisation.
> 

> > MITRE’s role as Secretariat for CVE program plays a key role to provide a resource to boost supply chain resilience. While its recent publicity and uncertainty in funding are regrettable, we are pleased that this matter has now been partially resolved.  

Author: Proposed change to address @rjb4standards's [comment](https://github.com/orcwg/orcwg/pull/127#discussion_r2151945494)

```suggestion
> MITRE’s role as Secretariat for CVE program plays a key role to provide a resource to boost supply chain resilience. While its recent publicity and uncertainty in funding are regrettable, we are pleased that this matter has been temporarily resolved.  
```

> 

> > The momentary alarm did however serve as a timely reminder to both governments and industry as to CVE’s global importance for supply chain resilience and need for wider global support.

Reviewer 4: Maybe rephrase "as to the global importance of a free, distributed and trusted vulnerability intelligence system for supply"

Author: ```suggestion
> The momentary alarm did however serve as a timely reminder to both governments and industry as to the global importance of a free, distributed and trusted vulnerability intelligence system for supply chain resilience and need for wider global support.
```

> 
> Indeed, this has also been recognised by several European governments who subsequently confirmed their wish to support (technically and financially) and thus we would hope that this support can be mirrored at an EU level, perhaps via ENISA, into a system which is mirrored infrastructure in Europe and co-developed.
> 
> We also see the growing role of national Computer Security Incident Response Team (CSIRTs), such as the National Cyber Security Centre (NCSC) in Ireland, largely supporting bug bounty programs, Capture the Flag (CTF) and other activities with local academia, government and businesses, investing in vulnerability discovery and management, and thus it is important to double down on the CVE program in a uniformed, cross-European and international way.
> 

> > It would also be advisable for the European Commission to include this need for reinforced collaboration in their next inter-institutional dialogue with USA CISA.

Reviewer 4: We should possibly also add the UK counterpart here.

Author: Do you want to make it more generic and include CISA and others as examples?

Reviewer 4: The UK has "National Cyber Security Centre"

> 

> > While we recognise the importance of NIS2’s European Vulnerability Database (EUVD) we feel there is still room for better alignment and co-ordination. Indeed, the EUVD’s beta version includes identifiers which are not aligned with CVEs or easily mapped.  This poses a significant risk to organisations and companies defending critical networks versus criminal or state sponsored hackers due to similar patterns but dissimilar meanings. This will create chaos for all producers and consumers of software bills of materials (SBOMs) and of vulnerability and exploitation data. It also increases the reporting burden on software producers and open source maintainers if they end up needing to apply for both CVE and EUVD assignments on each disclosed vulnerability.

Author: Combining @UlisesGascon's [suggestion](https://github.com/orcwg/orcwg/pull/127#discussion_r2149667961) with @fungi's and the original text. Let me know what you think.

```suggestion
>  While we recognise the importance of NIS2’s European Vulnerability Database (EUVD), we believe there is still room for better alignment and coordination. The EUVD beta includes identifiers that are not aligned or easily mapped to CVEs. This fragmentation creates confusion for producers and consumers of software bills of materials (SBOMs) and vulnerability data, reduces Europe’s ability to defend against criminal and state-sponsored threats, and increases compliance burdens and operational complexity for software teams, open source maintainers, and users. The European Commission should revise the CSA to ensure better alignment between CVE and EUVD systems.
```

Reviewer 5: Just one comment regarding "MITRE’s role as Secretariat for CVE program plays a key role to provide a resource to boost supply chain resilience. While its recent publicity and uncertainty in funding are regrettable, we are pleased that this matter has now been partially resolved.  "

The matter has been "temporarily" resolved. It's not clear if there will be a more permanent solution coming, but I believe we need an internationally supported permanent solution for software product vulnerability reporting to consumers.

Author: I like that reframing. Thank you.

Reviewer 3: Big +1

---
SIG: Cyber Resilience SIG
Document Type: Deliverable
Number: 2.6
Status: 🚀 Submitted
Group: ENISA
Date: 2025-06-14
Submitted: 2025-06-20
---

# Feedback to call for evidence on Cybersecurity Act Revision

## Context

Provide feedback to the [EU Commission][]'s [open call for evidence](https://ec.europa.eu/info/law/better-regulation/have-your-say/initiatives/14578-The-EU-Cybersecurity-Act_en) as part of its [initiative to revise the Cybersecurity Act (CSA)](https://digital-strategy.ec.europa.eu/en/news/commission-opens-consultation-revising-eu-cybersecurity-act). Urge for a stronger technical and strategic role for ENISA in light of the [recent funding issues of the US CVE program](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures#2025_funding_issues) and the increased importance of vulnerability handling in EU legislation.

Feedback provided on June 20, 2025 (reference: [F3567995](https://ec.europa.eu/info/law/better-regulation/have-your-say/initiatives/14578-The-EU-Cybersecurity-Act/F3567995_en)).

## Goals

* Urge the EU Commission to increase ENISA's mandate regarding CVE handling
* Avoid disproportionate reliance on U.S. government funding for CVEs

> ## Draft feedback

Author: ```suggestion
## Feedback
```

> [!NOTE]

> > Feedback must be provided in a comment box (so no formatting option). Text length is limited to 4,000 characters.

Author: ```suggestion
> ~~Feedback must be provided in a comment box (so no formatting option). Text length is limited to 4,000 characters.~~
```

> 

Author: ```suggestion
- [Feedback (PDF)](./deliverable-2-6-orcwg-csa-revision-feedback.pdf)

```
